### PR TITLE
[ko] fix: remove duplicate headings

### PR DESCRIPTION
**Pull request by yin1999**
### Description

remove duplicate headings. Since I don't speak Korean, please feel free to correct my mistakes.

### Motivation

Prepare for enabling [the `no-duplicate-heading` markdown-lint rule](https://github.com/mdn/translated-content/blob/d13c47b2b819b24e7623bb55ad5f24606c537d87/.markdownlint.jsonc#L20-L22).
Part of #17582

### CODE REVIEW

File: files/ko/web/api/element/scrollheight/index.md [hunk 1]
```diff
@@ -21,18 +21,6 @@ var intElemScrollHeight = element.scrollHeight;
...
-padding-top
-
-Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
-
-Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-
-padding-bottom
-
-**Left** **Top** **Right** **Bottom** _margin-top_ _margin-bottom_ _border-top_ _border-bottom_
-
```
Comment (yin1999) [inline files/ko/web/api/element/scrollheight/index.md:35]: Temporarily removed as this part is not relevant

File: files/ko/web/api/web_audio_api/simple_synth/index.md [hunk 2]
```diff
@@ -510,7 +510,7 @@ function playTone(freq) {
 
 oscillator의 주파수는 {{domxref("Oscillator.frequency")}} {{domxref("AudioParam")}} 객체의 값을 설정함으로써 `freq` 파라미터에 명시된 값으로 설정됩니다. 그리고서, 마침내, oscillator는 상속된 {{domxref("AudioScheduledSourceNode.start()")}} 메서드를 호출하여 소리를 생성하도록 시작됩니다.
 
-#### 음 재생하기
+#### 음을 연주하다
```
Comment (yin1999) [inline files/ko/web/api/web_audio_api/simple_synth/index.md:513]: Please check the translation, I'm not sure

File: files/ko/web/css/visual_formatting_model/index.md [hunk 3]
```diff
@@ -47,7 +47,7 @@ CSS 선택기는 해당 상자에 이름을 부여하거나 스타일링을 할
 
 상자를 포함하는 블록은 인라인수준 상자 또는 블록수준 상자만을 포함합니다. 그러나 문서는 두 가지 모두를 혼합해 포함합니다. 그 경우 무명 블록 상자는 인접 인라인수준 상자 주변에 생성됩니다.
 
-### 예제
+### 예제 1
```
Comment (yin1999) [inline files/ko/web/css/visual_formatting_model/index.md:50]: Use numbers to differentiate

Comment (hochan222) [reply files/ko/web/css/visual_formatting_model/index.md]: - https://github.com/mdn/translated-content/pull/17716/commits/f99f49cd99791ab3f058aad13525eb24ad274b9a

File: files/ko/web/api/element/scrollheight/index.md (hunk 1 above)
Comment (hochan222) [reply files/ko/web/api/element/scrollheight/index.md:35]: - https://github.com/mdn/translated-content/pull/17716/commits/37dd697c6bd5ce59afeac93b93b50008640fa27f

File: files/ko/web/api/web_audio_api/simple_synth/index.md (hunk 2 above)
Comment (hochan222) [reply files/ko/web/api/web_audio_api/simple_synth/index.md]: - https://github.com/mdn/translated-content/pull/17716/commits/37dd697c6bd5ce59afeac93b93b50008640fa27f

**Review by hochan222 — APPROVED** [review]
@yin1999 Thank you. Have a nice day :)

**Pull request merged**